packages/p2p/src/components/buy-sell/buy-sell-modal.jsx

```diff
@@ -86,6 +89,25 @@ const generateModalTitle = (formik_ref, my_profile_store, table_type, selected_a
     return localize('Sell {{ currency }}', { currency: selected_ad.account_currency });
 };
 
+const MarketRateChangeErrorModal = ({ is_open, closeModal, setShouldShowPopup }) => (
+    <Modal className='rate-changed-modal' is_open={is_open} onExited={() => setShouldShowPopup(true)} small>
+        <Modal.Body>
+            <Text
+                as='p'
+                align='left'
```

Comment:
we can remove align here since the default is `left`

Reply:
Got it!